prow/plugins/cla/cla.go

```diff
@@ -93,20 +92,20 @@ type gitHubClient interface {
 }
 
 func handleStatusEvent(pc plugins.Agent, se github.StatusEvent) error {
-	return handle(pc.GitHubClient, pc.Logger, se)
+	return handle(pc.GitHubClient, pc.Logger, se, pc.PluginConfig.CLAConfig)
 }
 
-// 1. Check that the status event received from the webhook is for the CNCF-CLA.
+// 1. Check that the status event received from the webhook is for the CNCF-CLA. The valid values are cla/linuxfoundation and EasyCLA.
```

Comment:
If these are the only valid values, I would add validation logic confirming as much.  Otherwise, this is a configuration-specific detail

Reply:
@mrbobbytables 
`The valid values are cla/linuxfoundation and EasyCLA` Is this a right statement to make?

Reply:
The comment "the valid values..." does not match the state of this code.  You're not validating that these are the only two values allowed.  _Any_ values are allowed as configuration data.  This plugin could just as easily be configured to apply the CLA label if the the `monkeys` status context was successful.  I think this is great.  Generally, more configurability for re-use is good.

If you agree with this, then you should drop the "the valid values" part of the comment.  But, If you disagree, and think this should be purpose-built for just those two status contexts, then you should validate that these are the only valid values in some kind of `validate` function.

Reply:
Okay I get it now. I am going to remove the comment that those are the only two valid values.